<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>894</width>
    <height>431</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Image Analysis</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color: black;
font: 75 italic 13pt &quot;PT Sans&quot;;
line-height: 1.3em;
background-color: rgb(40, 40, 40)
</string>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>401</width>
      <height>341</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="Line" name="line_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true">color:white</string>
       </property>
       <property name="text">
        <string>Метрики качества изображений:</string>
       </property>
       <property name="textFormat">
        <enum>Qt::AutoText</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="Line" name="line_5">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit">
         <property name="styleSheet">
          <string notr="true">color:white;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_14">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="browse">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>30</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">  background-color: rgb(59, 58, 59);
  border: none;
  color: white;
  border-radius: 5px</string>
         </property>
         <property name="text">
          <string>Обзор</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_13">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line_12">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QPushButton" name="save">
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">  background-color: rgb(59, 58, 59);
  border: none;
  color: white;
  border-radius: 5px</string>
         </property>
         <property name="text">
          <string>Сохранить</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="Line" name="line_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="average">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>10</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>30</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">  background-color: rgb(59, 58, 59);
  border: none;
  color: white;
  border-radius: 5px</string>
         </property>
         <property name="text">
          <string>Метод усреднения</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="firstblur">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>10</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>30</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">  background-color: rgb(59, 58, 59);
  border: none;
  color: white;
  border-radius: 5px</string>
         </property>
         <property name="text">
          <string>Мера размытости №1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_10">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line_7">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="Line" name="line_9">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="secondblur">
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>30</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">  background-color: rgb(59, 58, 59);
  border: none;
  color: white;
  border-radius: 5px</string>
         </property>
         <property name="text">
          <string>Мера размытости №2</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_6">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="enthropy">
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>30</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">  background-color: rgb(59, 58, 59);
  border: none;
  color: white;
  border-radius: 5px</string>
         </property>
         <property name="text">
          <string>Мера энтропии</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_11">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line_15">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="Line" name="line_17">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="segmentation">
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>30</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">  background-color: rgb(59, 58, 59);
  border: none;
  color: white;
  border-radius: 5px</string>
         </property>
         <property name="text">
          <string>Мера сегментации</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_16">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="sharpness">
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>30</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">  background-color: rgb(59, 58, 59);
  border: none;
  color: white;
  border-radius: 5px</string>
         </property>
         <property name="text">
          <string>Мера резкости №1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_18">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line_8">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="Line" name="line_19">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="sharpness2">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>20</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">  background-color: rgb(59, 58, 59);
  border: none;
  color: white;
  border-radius: 5px</string>
         </property>
         <property name="text">
          <string>Мера резкости №2</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_20">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="exit">
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>20</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">  background-color: rgb(59, 58, 59);
  border: none;
  color: white;
  border-radius: 5px</string>
         </property>
         <property name="text">
          <string>Выход</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_21">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>321</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white</string>
    </property>
    <property name="text">
     <string> Copyright © Parviz Jamilov (All rights reserved)</string>
    </property>
   </widget>
   <widget class="QLabel" name="img">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>10</y>
      <width>451</width>
      <height>351</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>hi.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>894</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="separator"/>
    <addaction name="action_2"/>
    <addaction name="separator"/>
    <addaction name="action_5"/>
    <addaction name="separator"/>
    <addaction name="action_7"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Помощь</string>
    </property>
    <addaction name="action_8"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_2">
   <property name="text">
    <string>Открыть</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>Сохранить</string>
   </property>
  </action>
  <action name="action_7">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="action_8">
   <property name="text">
    <string>Все просто!</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
